ts/store/reducers/identification.ts

```diff
@@ -1,17 +1,27 @@
 import { getType } from "typesafe-actions";
 
+import { fromNullable } from "fp-ts/lib/Option";
+import { PersistPartial } from "redux-persist";
 import { PinString } from "../../types/PinString";
 import {
   identificationCancel,
+  identificationFailure,
   identificationReset,
   identificationStart,
   identificationSuccess
 } from "../actions/identification";
 import { Action } from "../actions/types";
 
+const freeAttempts = 4;
+// in seconds
+const deltaTimespanBetweenAttempts = 30;
```

Comment:
```suggestion
const deltaTimespanBetweenAttempts = 30 as Second;
```
we could avoid comment if we have types that talk about what they mean